CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog]
 ## [Unreleased]
 
 - Allow claimants to search by school postcode
+- Replace the boolean "has this claim passed its check deadline" which is sent
+  to Geckoboard with a string representing the state of the claim relative to
+  the deadline
```

Comment:
I think the aim is to keep the changelog as something that everyone (including non-devs) can read, this might be too technical? Maybe we could use the PR title `Add "approaching SLA" state to Geckoboard dataset` or `Replace "passed SLA" with "approaching SLA" state to Geckoboard dataset`?